packages/hashi/webpack.config.bloom.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
```

Comment:
This webpack config doesn't seem to be needed - everything you are doing in here could be done with a simple NodeJS script without using webpack at all.

I had assumed you would need webpack to bundle the bloom player dist and any other files you needed, rather than injecting them as individual JS files into the existing HTML.